app/src/main/kotlin/com/hedvig/app/ApplicationModule.kt

```diff
@@ -296,26 +283,28 @@ val apolloClientModule = module {
   }
 }
 
-fun makeUserAgent(locale: Locale) =
-  "${
-    BuildConfig.APPLICATION_ID
-  } ${
-    BuildConfig.VERSION_NAME
-  } (Android ${
-    Build.VERSION.RELEASE
-  }; ${
-    Build.BRAND
-  } ${
-    Build.MODEL
-  }; ${
-    Build.DEVICE
-  }; ${
-    locale.language
-  })"
+fun makeUserAgent(locale: Locale): String = buildString {
+  append(BuildConfig.APPLICATION_ID)
+  append(" ")
+  append(BuildConfig.VERSION_NAME)
+  append(" ")
+  append("(Android")
+  append(" ")
+  append(Build.VERSION.RELEASE)
+  append("; ")
+  append(Build.BRAND)
+  append(" ")
+  append(Build.MODEL)
+  append("; ")
+  append(Build.DEVICE)
+  append("; ")
+  append(locale.language)
+  append(")")
+}
```

Comment:
I think even ktlint was changing this all the time, so moved it to a `buildString` call to never have formatting problems again 😅